google/cloud/logging_v2/gapic/config_service_v2_client.py

```diff
@@ -74,58 +74,73 @@ def from_service_account_file(cls, filename, *args, **kwargs):
     from_service_account_json = from_service_account_file
 
     @classmethod
-    def billing_path(cls, billing_account):
-        """Return a fully-qualified billing string."""
+    def billing_account_path(cls, billing_account):
+        """Return a fully-qualified billing_account string."""
         return google.api_core.path_template.expand(
             "billingAccounts/{billing_account}", billing_account=billing_account,
         )
```

Comment:
Wow, this is a huge set of potentially breaking changes.

Reply:
Yep, lots of breaking changes. I looked at previous iterations of this PR and basically decided it was too much to patch over with synth replaces (path helper removals, path helper renames, method params being reordered).

@simonz130 I'd prefer to continue to let this sit and take the breaking changes when this library moves to the microgenerator. Is that alright with you? I'll also need to sync with you at some point about what to do with the handwritten layer.